onnxruntime/core/session/inference_session.cc

```diff
@@ -292,6 +293,21 @@ void InferenceSession::ConstructorCommon(const SessionOptions& session_options,
   // a monotonically increasing session id for use in telemetry
   session_id_ = global_session_id_.fetch_add(1);
   allocator_manager_ = std::make_shared<onnxruntime::AllocatorManager>();
+
+  // Add log to allow serving platforms to quantify ORT usage.
+  // To avoid flooding the test logs, this is done for non-debug mode only
+  // TODO: plug-in a platform specific telemetry provider to send the telemetry to
+#ifdef NDEBUG
+  std::ostringstream ostr;
+  // Format: "ORT Telemetry: Version = 1.7.0; Event = EventName (event_attr1: foo.onnx, event_attr2: 400us)"
+  // Format: "ORT Telemetry: Version = 1.7.0; Event = SessionCreation (model: foo.onnx, ts: 400us)"
+  ostr << "ORT Telemetry: "
+       << "Ver = " << ORT_VERSION << "; Event = SessionCreation";
```

Comment:
Does this need to match the format in the comments and be 'Version' instead of 'Ver'?